middlewares/validators/validator.helper.js

```diff
@@ -41,7 +41,7 @@ function integerValidator(
                 return value >= lowerBound && value <= upperBound;
             })
             .withMessage(
-                `${fieldname} must be between ${lowerBound} and  ${upperBound}`
+                `${fieldname} must be between ${lowerBound} and ${upperBound}`
```

Comment:
does this only validate that travel is an integer? or does it also cover the [0,100] inclusive bound?

Reply:
I make a new test all together that also validates the 0-100 bound

Reply:
does it validate float vs int? or is that being done on front end?

Reply:
@Ty-Won that would be a frontend validation, I'll have a PR for that tn 😉

Reply:
If we want to stay properly RESTful the int validation should be on the API

Reply:
Agreed we should have api validation

Reply:
The validator does check .isInt() which seems to work. If I input a float, like 10.5 for travel, then indeed the error appears saying that it should be an integer. Do we need more than this?

Reply:
Seems good to me, I also added int validation on the frontend: https://github.com/hackmcgill/dashboard/pull/764